### PR TITLE
Improved README

### PR DESCRIPTION
**Pull request by fzliu**
- Increased emphasis on anything-to-vec.
- Removed unnecessary architecture details.
- Added Slack and other badge links.
- Improved overall readability.

### CODE REVIEW

**Comment by shiyu22** [comment]
I think it should be updated in Getting started with embedding:
```bash
>>> from towhee import pipeline

# Use our in-built embedding pipeline
>>> img_path = 'towhee_logo.png'
>>> embedding_pipeline = pipeline('image-embedding')
>>> embedding = embedding_pipeline(img_path)
```
Because the input of the pipeline has been changed to str (image path).

**Comment by fzliu** [comment]
@filip-halt @shiyu22 Updated the README according to the above comments. In the future, please add line-based comments within the commit instead of PR-level comments.

File: README.md [hunk 1]
```diff
@@ -1,75 +1,85 @@
...
 
-Custom machine learning pipelines can be defined in a YAML file and uploaded to the Towhee hub (coming soon&trade;). Pipelines which already exist in the local Towhee cache (`/$HOME/.towhee/pipelines`) will be automatically loaded:
+## Dive deeper
+
+Custom machine learning pipelines can be defined in a YAML file or via a Spark-like high-level programming interface (coming soon &trade;). The first time you instantiate and use a pipeline, all Python functions, configuration files, and model weights are automatically downloaded from the Towhee hub. To ease the development process, pipelines which already exist in the local Towhee cache (`/$HOME/.towhee/pipelines`) will be automatically loaded:
 
 ```python
-# This will load the pipeline defined at $HOME/.towhee/pipelines/fzliu/resnet50_embedding.yaml
->>> embedding_pipeline = pipeline('fzliu/resnet50_embedding')
->>> embedding = embedding_pipeline(img)
```
Comment (shiyu22) [inline README.md:56]: It should also be updated to `embedding = embedding_pipeline(img_path)`.

File: README.md [hunk 2]
```diff
@@ -1,75 +1,85 @@
...
+[![Github Actions](https://img.shields.io/github/workflow/status/towhee-io/towhee/Workflow%20for%20pylint/main?label=pylint&style=for-the-badge)]()
+[![Coverage](https://img.shields.io/codecov/c/github/towhee-io/towhee?style=for-the-badge)]()
 
 ## What is Towhee?
 
-Towhee is a flexible machine learning framework currently focused on computing deep learning embeddings over unstructured data. Built on top of PyTorch and Tensorflow (coming soon&trade;), Towhee provides a unified framework for running machine learning pipelines locally, on a multi-GPU/TPU/FPGA machine (coming soon&trade;), or in the cloud (coming soon&trade;). Towhee aims to make democratize machine learning, allowing everyone - from beginner developers to AI/ML research groups to large organizations - to train and deploy machine learning models.
+Towhee is an flexible, application-oriented framework for computing embedding vectors over unstructured data. It aims to make democratize `anything2vec`, allowing everyone - from beginner developers to large organizations - to train and deploy complex machine learning pipelines with just a few lines of code.
```
Comment (filip-halt) [inline README.md:15]: 'aims to make democratize' sounds wrong. Id just do 'democratize'

Comment (filip-halt) [reply README.md:15]: everyone -> anyone, but that is just personal writing style

File: README.md [hunk 3]
```diff
@@ -1,75 +1,85 @@
...
+
+Towhee has pre-built pipelines for a variety of tasks, including audio/music embeddings, image embeddings, celebrity recognition, and more. For a full list of pipelines, feel free to visit our [Towhee hub](https://hub.towhee.io).
 
 ## Key features
 
 - __Easy embedding for everyone__: Transform your data into vectors with less than five lines of code.
```
Comment (filip-halt) [inline README.md:21]: with less than five -> in less than five
